backend/src/test/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceControllerTest.java

```diff
@@ -29,12 +30,14 @@
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.context.annotation.Import;
 import org.springframework.http.MediaType;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
 
 @AutoConfigureMockMvc
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@Import(TestIdentityConfiguration.class)
```

Comment:
"Test" vs "Demo" vs "Dev" naming is super confusing sometimes. I see a new file `DemoUserIdentitySupplier.java`

Reply:
There is a long-term plan underlying those names, though this one challenged me. The goal for `DemoUserIdentitySupplier` (and accompanying new things coming soon) is to have the `DemoUserConfiguration` and its underlying list of `DemoUser` objects be the thing that drives the user authn/authz for the application in no-security mode. This configuration (which should have a javadoc explaining this next bit) is to apply that paradigm to controller (and eventually API) tests, which currently use the default no-security setup with a bunch of Mockito thrown on top to make it more or less work.

I'm totally happy to look at suggestions to rename either this configuration or any and all of the `Demo${THING}` classes if there is a self-consistent approach that makes more sense to people who aren't me: this is a very path-dependent set of names we've landed on, and we can make intentional choices to fix that. 😸

Reply:
Yeah, I don't have a better suggestion. It's always just been a point of confusion for me. For this feature, it would be super useful for Dev setups to default to it so we can test different types of users quickly.

I'm going to mark this item resolved since I can't suggest better solution.